docs/src/cluster/turbine-block-propagation.md

```diff
@@ -2,27 +2,27 @@
 title: Turbine Block Propagation
 ---
 
-A Solana cluster uses a multi-layer block propagation mechanism called _Turbine_ to broadcast transaction shreds to all nodes with minimal amount of duplicate messages. The cluster divides itself into small collections of nodes, called _neighborhoods_. Each node is responsible for sharing any data it receives with the other nodes in its neighborhood, as well as propagating the data on to a small set of nodes in other neighborhoods. This way each node only has to communicate with a small number of nodes.
+A Solana cluster uses a multi-layer block propagation mechanism called _Turbine_ to broadcast transaction shreds to all nodes with minimal amount of duplicate messages. The cluster divides itself into small collections of nodes, called _neighborhoods_. Each node is responsible for propagating any data it receives on to a small set of nodes in downstream neighborhoods and possibly sharing data with the other nodes in its neighborhood. This way each node only has to communicate with a small number of nodes.
 
 During its slot, the leader node distributes shreds between the validator nodes in the first neighborhood \(layer 0\). Each validator shares its data within its neighborhood, but also retransmits the shreds to one node in some neighborhoods in the next layer \(layer 1\). The layer-1 nodes each share their data with their neighborhood peers, and retransmit to nodes in the next layer, etc, until all nodes in the cluster have received all the shreds.
 
 ## Neighborhood Assignment - Weighted Selection
 
 In order for data plane fanout to work, the entire cluster must agree on how the cluster is divided into neighborhoods. To achieve this, all the recognized validator nodes \(the TVU peers\) are sorted by stake and stored in a list. This list is then indexed in different ways to figure out neighborhood boundaries and retransmit peers. For example, the leader will simply select the first nodes to make up layer 0. These will automatically be the highest stake holders, allowing the heaviest votes to come back to the leader first. Layer 0 and lower-layer nodes use the same logic to find their neighbors and next layer peers.
 
-To reduce the possibility of attack vectors, each shred is transmitted over a random tree of neighborhoods. Each node uses the same set of nodes representing the cluster. A random tree is generated from the set for each shred using a seed derived from the leader id, slot and shred index.
+To reduce the possibility of attack vectors, each shred is transmitted over a random tree of neighborhoods. Each node uses the same set of nodes representing the cluster. A random tree is generated from the set for each shred using a seed derived from the leader id, slot, shred index, and shred type.
 
 ## Layer and Neighborhood Structure
```

Comment:
I think it would be helpful to link or include this plot:
https://discord.com/channels/428295358100013066/478692221441409024/1006520189917081732

Reply:
Agree. A picture's worth 1000 words. Any idea how to upload images?

Reply:
Like this?
https://github.com/solana-labs/solana/blob/180ea1e0a/docs/src/cluster/stake-delegation-and-rewards.md?plain=1#L134

Reply:
Right, but how do we access the web host to push images into that `/img` directory?

Reply:
Can't we just add the png file here?
https://github.com/solana-labs/solana/tree/180ea1e0a/docs/static/img

Reply:
If that works, that's simple enough. I'm a little skeptical because I don't see the existing turbine images in that directory:
```
/img/data-plane-seeding.svg
/img/data-plane-fanout.svg
/img/data-plane.svg
/img/data-plane-neighborhood.svg
```

Reply:
yeah those are generated from bob files:
https://github.com/solana-labs/solana/tree/180ea1e0a/docs/art
https://github.com/solana-labs/solana/blob/180ea1e0a/docs/convert-ascii-to-svg.sh

Reply:
Ahhh I see. Thanks!